sptx_format/cli.py

```diff
@@ -32,14 +32,65 @@ def validate(ctx, experiment_json, fuzz):
 
 
 @validate.command()
-@click.argument("experiment_json", metavar="JSON_FILE_OR_URL")
+@click.argument("path", metavar="JSON_FILE_OR_URL")
 @click.option("--fuzz", is_flag=True)
 @click.pass_context
-def experiment(ctx, experiment_json, fuzz):
+def codebook(ctx, path, fuzz):
+    """validate codebook against the jsonschemas"""
+
+    try:
+        valid = validate_sptx.validate_file(path, "codebook/codebook.json", fuzz)
+        if valid:
+            ctx.exit(0)
+        else:
+            ctx.exit(1)
+    except KeyboardInterrupt:
+        ctx.exit(3)
+
+
+@validate.command()
+@click.argument("path", metavar="JSON_FILE_OR_URL")
+@click.option("--fuzz", is_flag=True)
+@click.pass_context
+def experiment(ctx, path, fuzz):
     """validate experiment against the jsonschemas"""
 
     try:
-        valid = validate_sptx.validate(experiment_json, fuzz)
+        valid = validate_sptx.validate(path, fuzz)
+        if valid:
+            ctx.exit(0)
+        else:
+            ctx.exit(1)
+    except KeyboardInterrupt:
+        ctx.exit(3)
+
+
+@validate.command()
+@click.argument("file", metavar="JSON_FILE_OR_URL")
+@click.option("--fuzz", is_flag=True)
+@click.pass_context
+def fov(ctx, file, fuzz):
+    """validate field-of-view against the jsonschemas"""
+
+    try:
+        valid = validate_sptx.validate_file(file, "field_of_view/field_of_view.json", fuzz)
```

Comment:
tiny nit: should avoid using python keywords as variable names.

Reply:
https://github.com/spacetx/starfish/pull/1062